funnel/templates/proposal.html

```diff
@@ -2,6 +2,7 @@
 {% from "baseframe/forms.html" import renderfield, renderform %}
 {% from "comments.html" import commenttree %}
 {% block title %}{{ proposal.title }} &mdash; {{ space.title }}{% endblock %}
+{% block description %}{{ proposal.title }}{% if proposal.speaker %}{% trans %} by {% endtrans%}{{ proposal.speaker.pickername }}{% endif %}, {{ space.title }}{% endblock %}
```

Comment:
The `{% trans %}` block has to be for the entire string. You can't assume all languages will have an equivalent for "by" that appears before the speaker's name.

However, you can't have an `{% if %}` inside it, so you have to move the `if` block out as well and repeat `{% proposal.title %}` under both `if` and `else` conditions.